cybersecuritytools/csls/generate_cloudwatch_logs/query_splunk.py

```diff
@@ -0,0 +1,65 @@
+import os
+from typing import Any, Dict, List
+
+from .put_cloudwatch_logs import CloudWatchLogResult, log_formats, log_group_name
+
+
+def log_group_name_to_splunk_format(group_name: str) -> str:
+    """Transform the standard Cloudwatch log group name to the format used
+    by Splunk.
+
+    /aws/foo/bar -> aws:foo:bar
+    """
+    return group_name.replace("/", ":").strip(":")
+
+
+# TODO: Search for the exact cloudwatch log payload instead of this wide search.
+def search_query(test_type: str, index: str = "test_data") -> str:
+    """Generate search query to find the test data."""
+    sourcetypes = ", ".join(
+        [log_group_name_to_splunk_format(log_group_name(f)) for f in log_formats()]
+    )
+    uuid = 80000
```

Comment:
80000 isn't in uuid format, I think. How about `80000000-000-0000-0000-000000000000`?

Reply:
I've changed this to a real UUID.